src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-font-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-font-color;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-text {
+      display: block;
+      padding-top: 1px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $main-font-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    gap: 85px;
+
+    &-word {
+      display: inline-block;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-font-color;
+    }
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-font-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 166px;
```

Comment:
Don't fix container size (if there is no such a requirement)
<img width="558" alt="image" src="https://github.com/user-attachments/assets/623bc860-db1c-43b6-8fbf-256b9e0059b1">

Reply:
This is not in the requirements and in this case i made a width for button

Reply:
> This is not in the requirements and in this case i made a width for button

Yes, that why you can use just `100%` instead because in case the card width changes your button will have the wrong width